### ISSUE TITLE
Context menu in "Git Repositories" window shows only "show in" entry with EasyShell installed

### ISSUE THREAD

**Issue by simon-hyggeonkel** [anb0s/EasyShell#195]
## Expected Behavior
when right clicking a repository, or a specific branch, I expect to see a long list of options such as pull and checkout.

## Actual Behavior
What I am presented with is only the "Show In" menu entry.

## Steps to Reproduce the Problem
Install EasyShell plugin V. 2.2.0
Switch to Perspective "Git"
Right click a repository in the "Git Repositories" window.

## Your Environment
- Plugin version: 2.2.0
- Eclipse Version: Version: 2021-03 (4.19.0) Build id: 20210312-0638
- Java Version: 15.0.1 64 bit
- Operating System Version: Windows 10 Pro v 20H2

**Comment by simon-hyggeonkel** [comment]
[log.txt](https://github.com/anb0s/EasyShell/files/6520502/log.txt)

Here is my log.

**Comment by anb0s** [comment]
Thanks, I will try to reproduce. Which eclipse distribution are you using?

P.S.: I will be available after 07.06.21 so it will took some time...

**Comment by tomaswolf** [comment]
Just for the record: this was originally reported in the [Egit forum](https://www.eclipse.org/forums/index.php/t/1108003/), including a [screenshot](https://www.eclipse.org/forums/index.php?t=getfile&id=40512).

**Comment by simon-hyggeonkel** [comment]
Yes, that was me posting the problem in the wrong place. :)

**Comment by anb0s** [comment]
I've investigated but cannot reproduce with standard Eclipse distributions without CDT (Java, PDE etc. only). Again the question from above:

**Which eclipse distribution are you using?**

It looks like the CDT bundle is found, but the class **IncludeReferenceProxy** cannot be loaded...

**Comment by tomaswolf** [comment]
@anb0s : according to the log he posted:
```
Framework arguments:  -product org.eclipse.epp.package.php.product
```
Eclipse PHP includes org.eclipse.cdt.core, but only that. It does not include UI bundles from CDT.

`ResourceUtilsCDT` checks for two classes from CDT UI, but is guarded only by a check for CDT core.

**Comment by anb0s** [comment]
@tomaswolf : ah yes, thank you! I will extend the guard for ui bundle and add catch for all exceptions as this should not throw and not corrupt the menu propagation

**Comment by anb0s** [comment]
Please try the testing version:  2.2.1.v20210607-1525
It is available at the testing update site now: http://anb0s.github.io/EasyShell/testing

**Comment by tomaswolf** [comment]
That solves the problem.

1. Downloaded Eclipse PHP (2021-03).
2. Opened the Git Repositories view, imported an existing project, verified all menu entries present.
3. Installed EasyShell from Marketplace: menu entries in Git Repositories view are gone.
4. Installed EasyShell 2.2.1 from the above testing update site: menus are back.

**Comment by anb0s** [comment]
@tomaswolf Thanks! I have tested similar installation and it works for me too.
@simon-hyggeonkel new version will be released soon please use testing until

**Comment by anb0s** [comment]
Please use released version: EasyShell v2.2: 2.2.1.v20210716-0745
